.github/workflows/nightly-pekko-1.1-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: Nightly Testing with Pekko 1.1
+
+permissions: {}
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Should we defer the time after pekko nightly published？

Reply:
makes sense - I have changed this to run at 4am